config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Encass.java

```diff
@@ -202,7 +202,8 @@ public void setAnnotatedEntity(AnnotatedEntity<Encass> annotatedEntity) {
     public String getType() {
         return EntityTypes.ENCAPSULATED_ASSERTION_TYPE;
     }
-    
+
+    @JsonIgnore
```

Comment:
getShortenedType is already ignored at AnnotableEntity

Reply:
That's not working. We must use JsonIgnore at the top-level class definition.